flow/connectors/utils/array.go

```diff
@@ -16,3 +16,16 @@ func ArrayMinus(first []string, second []string) []string {
 	}
 	return result
 }
+
+func ArrayPartition[T any](slice []T, size int) [][]T {
```

Comment:
This is generally referred to as chunking; partition tends to imply some key to group by